stackslib/src/chainstate/stacks/db/blocks.rs

```diff
@@ -5291,6 +5335,7 @@ impl StacksChainState {
                     cur_epoch.start_height,
                 )
             }
+            StacksEpochId::Epoch30 => todo!(), // Need input here
```

Comment:
This should share the behavior of 2.1-2.4.

Reply:
Okay, moved into their match arm